addon/components/polaris-resource-list/item.js

```diff
@@ -221,6 +222,7 @@ export default Component.extend(context.ConsumerMixin, {
       'selectMode',
       'element'
     );
+    let { ctrlKey, metaKey } = event;
```

Comment:
should this be `event.nativeEvent`?

Reply:
No, React wraps all event objects in a [`SyntheticEvent`](https://reactjs.org/docs/events.html) class, whose `nativeEvent` property is the underlying browser event. We don't seem to have to worry about such things in Ember-land 🙌

Reply:
sweet 🙌